src/west/commands/__init__.py

```diff
@@ -45,8 +45,9 @@ def __init__(self, **kwargs):
         self.hint = kwargs.pop('hint', None)
         super(ExtensionCommandError, self).__init__(**kwargs)
 
-_NO_TOPDIR_MSG_FMT = '''\
-no west installation found from "{}"; "west {}" requires one.
+def _no_topdir_msg(cwd, name):
```

Comment:
Not sure why you defined a function instead of keeping the global with the `{cwd}` and `{name}` inside them. Isn't the f-string evaluated whenever you actually use it? or does that happen when you define it?

Reply:
To be honest, I don't know if I could have done this with an f-string, but I don't think so because the format arguments aren't in global scope, so I put them on the function stack to make them available.